app/views/content/covid_19_guidance.md

```diff
@@ -0,0 +1,54 @@
+* To give you more time to make decisions, we won’t reject or decline any offers automatically until 20 April 2020. Below we outline what this means for you and your candidates. We’ll keep you updated if anything else changes. 
+
+## Giving you more time before applications are rejected by default (RBD)
+
+#### Applications that have been submitted already
+
+You have until 23:59 on 20 April to make decisions on applications which would normally be automatically rejected on or before 20 April.
+
+For applications which have a RBD date after 20 April, you’ll also have more time to make a decision. We’ll add an extra 4 weeks to the response deadlines.
+
+#### Applications that are submitted between now and 20 April
+
+If applications come in between now and 20 April, you have until 23:59 on 15 June to respond. No applications in this category will be automatically rejected until then.
+
+#### How to keep track of your new RBD dates
+
+You can see your new RBD dates in [Manage teacher training applications](/provider/applications).
+
+#### Keeping candidates up to date
+
+We’ll let candidates know that it might take you longer to respond to their application, and tell them your new RBD date. 
+
+## Giving candidates more time to make decisions on offers
+
+#### Hearing back from candidates might take longer
+
+We’re pausing all automatic decline by default (DBD) dates until 20 April. This means that there may be a delay hearing back from any candidates who would normally have to respond to an offer on or before 20 April. We’re giving everyone in this category until 23:59 on 20 April to respond. 
+
+There may also be a delay hearing back from candidates who have a DBD date after 20 April. For example, if you made the offer before 20 April, a candidate will have a bit more time than the usual 10 days to respond because we’re pausing the system until 20 April. 
+
+#### How to keep track of candidates’ new DBD dates
+
+You can find candidates’ new DBD dates in [Manage teacher training applications](/provider/applications).
+
+#### Keeping candidates up to date
+
+We’ll let candidates know if they have longer to respond to an offer and tell them their new DBD date.  
+
+## What we ask of you during this time
+
+#### Keep your vacancy statuses up to date
+
+Please update the statuses of your training programmes to tell potential applicants if you’re not recruiting at the moment.
+
+
+Providers in England can do this using [Find postgraduate teacher training courses](https://www2.find-postgraduate-teacher-training.service.gov.uk/).
+
+
+Providers in Wales can do this using [UCAS Search](https://digital.ucas.com/search).  
+
+## Get help 
+
+If you have any questions, you can contact your relationship manager or email us at [becomingateacher@education.gov.uk](mailto:becomingateacher@education.gov.uk)
```

Comment:
Can use shorter markdown syntax for emails: `<becomingateacher@education.gov.uk>`